package-versions/grouping-and-overrides/grouping-and-overrides.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: default
+title: Grouping and Overrides
+parent: Package Versions
+has_children: true
+nav_order: 2
+---
+
+# Grouping and Overrides
+
+Managing package versions across large numbers of complex projects may benefit from organizing `<PackageVersion>` into named groups of packages.
+
+Additionally, the usage of named groups of `<PackageVersion>` can allow individual projects to use different versions than the primary versions used globally.
+
+## Introduction
+
+In the most simple case, a `<PackageVersion>` assigned to a named group can be used to separate all of the versions in a project.
+
+#### Directory.Packages.props
+```xml
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <PackageVersion Include="foo" Version="1.1.1" />
+        <PackageVersion Include="foo" Version="2.2.2" CentralManagementGroup="A" />
```

Comment:
I wonder if `Group` would suffice, given the context is a `PackageVersion` item.